src/components/Discography/Discography.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const Discography = ({ artistName, releases }) => {
+  const releaseElements = releases.map(release => (
+    <li key={release.releaseId}>
+      <Link to={`/releases/${artistName}/${release.releaseId}`}>
+        <img src={`http://coverartarchive.org/release/${release.releaseId}/front`} onError={(e)=>{e.target.onerror = null; e.target.src = 'https://vassifer.blogs.com/.a/6a00d8341c18b253ef0191048c97fb970c-800wi';}}/>
```

Comment:
Agreed with Scott, although I like the use of onError. This might be too much url for one line.